tests/pathfinding/test_graphs.py

```diff
@@ -99,7 +99,7 @@ def test_routing_simple(
     view01: ChannelView = token_network_model.G[addresses[0]][addresses[1]]["view"]
     view10: ChannelView = token_network_model.G[addresses[1]][addresses[0]]["view"]
 
-    assert view01.deposit == 100
+    # assert view01.deposit == 100
```

Comment:
Why not remove that line?

Reply:
Classic, I was sure I would remember to remove this once the tests pass :-/